CIScripts/Build/BuildFunctions.ps1

```diff
@@ -294,21 +225,8 @@ function Invoke-ProductUnitTests {
             "timeout's threshold. They were copied from Linux unit test job.")]
         $Env:TASK_UTIL_RETRY_COUNT = 6000
 
-        $AgentExecutables = Get-ChildItem -Recurse (Join-Path $TestPathPrefix '*.exe') | `
-            Where-Object Directory -match '\\build\\.*\\test'
-
-        $TestRes = $AgentExecutables | ForEach-Object {
-            Invoke-ProductUnitTestRunner -TestExecutable $( $_.FullName )
-        }
-
-        $TestRes | ForEach-Object {
-            if (0 -ne $_) {
-                throw "Running tests failed"
-            }
-        }
-
-        $Env:Path = $backupPath
+        Invoke-NativeCommand -ScriptBlock {
+            scons -j $Env:BUILD_THREADS --opt=$BuildMode @Tests
+        } | Out-Null
```

Comment:
Won't the Out-Null prevent logs from being written?

Reply:
No